tests/rptest/tests/schema_registry_test.py

```diff
@@ -1223,6 +1223,21 @@ def move_leader():
                 check_connection(n.account.hostname)
             move_leader()
 
+    @cluster(num_nodes=3)
+    def test_redpanda_admin_restart_schema_store(self):
+        admin = Admin(self.redpanda)
+
+        try:
+            admin.redpanda_services_restart(rp_service='foobar')
```

Comment:
An additional test for empty service might be useful, I think it will return 400 (bad request).